### DISCUSSION TITLE
Table 19.5 "Lunar Phases" is a bit out of place in the User Guide

### DISCUSSION THREAD

**Discussion by adgorn** [Stellarium/stellarium#2930] [General]
It is up a bit to soon, in section 19.2.5.  See pg 269.  Likewise Fig 19.4 looks like it should be pushed down a bit too.  Alan

**Comment by gzotti** [comment]
Thank you for keen eyes 5 and 6 (I hope). However, they are on top of the same page where their respective section starts. LaTeX prefers moving float material to the top of the page. We could force it to within the paragraph or maybe bottom of the page, but I dislike that. I might try relocating (maybe I did already around 2016...), but will likely revert as it may look not better than now. This is just a free User Guide after all.

**Reply by adgorn** [reply]
I love the UG and truly appreciate you putting that together.  I've been a casual Stellarium user for years but this month decided to do a deep dive.  One motivation was that our local astronomy club had a presentation on Stellarium, which was excellent.  The Guide has been extremely helpful. Thank you very much for all the hard work on Stellarium!
![Astro club meeting on Stellarium](https://user-images.githubusercontent.com/24814109/209478105-53420e88-a799-46a5-bf41-20b684524640.jpg)
